tests/datasets/test_oscd.py

```diff
@@ -0,0 +1,9 @@
+from torchgeo.datasets import OSCD
```

Comment:
This should have the same structure as the other tests in this directory. We don't want to download the entire dataset to test it.

Reply:
Yup, I was just making small changes yesterday due to other activities. Sorry I shouldn't be pushing commits that I don't think are ready for review. Will keep this in mind in the future!

Reply:
You can also put the PR in "draft" mode to indicate that it's not ready for review.

Reply:
Forgot about that. Will do that now, thanks!